eitprocessing/features/breath_detection.py

```diff
@@ -48,9 +53,17 @@ class BreathDetection:
     invalid_data_removal_percentile: int = 5
     invalid_data_removal_multiplier: int = 4
 
-    def find_breaths(self, sequence: Sequence, continuousdata_label: str) -> IntervalData:
+    @singledispatchmethod
+    def find_breaths(self, container: Sequence | ContinuousData) -> IntervalData:
         """Find breaths in the data.
 
+        You can either pass a ContinuousData object as first argument, or a
+        Sequence as first argument and the label of a ContinuousData object.
+        When passing a ContinuousData object, optionally, you can pass a
+        Sequence object as second argument.
+        If a sequence is passed to `find_breaths()`, the resulting breaths
+        are stored in the Sequence as `intervaldata["breaths"]`.
```

Comment:
it's a bit inconsistent that "find_breaths()" is between backticks, while e.g. "Sequence" and "ContinuousData" are not.
I imagine that we may not have been very consistent with this throughout. I am currently unsure which of the two is best, and it might be something that needs to be tested to see how renders best in #250 .

Reply:
Fixed in 974b47363e66b42538809f091a2ef390fa731d86.